discover.go

```diff
@@ -63,7 +63,13 @@ func ensureDir(fp string) error {
 func cacheDBFile(cacheFilePath string) error {
 	ensureDir(cacheFilePath)
 
-	response, err := http.Get(PCIIDS_URI)
+	client := new(http.Client)
+	request, err := http.NewRequest("GET", PCIIDS_URI, nil)
+	if err != nil {
+		return err
+	}
+	request.Header.Set("User-Agent", "golang-jaypipes-pcidb")
```

Comment:
Mind throwing the "golang-jaypipes-pcidb" into a `const` string at the top of this file please @martinkennelly?

Reply:
My pleasure. done. Thank you for such a quick review!